### PR TITLE
Allow setting transmission range on wifi comp & sets a maximum range

### PR DESCRIPTION
**Pull request by glassofmilk**
[FEATURE]
## About the PR 
Allows for setting the transmission range on a wifi component. They were infinite range before - this lets you change it from anywhere under the max limit of 64

## Why's this needed? 
Now that portable mechcomp is a thing, being able to limit the range on a wifi comp is very useful.

## Changelog

```
(u)glassofmilk:
(+)Changed wifi component to allow setting a max transmission range.
```

### CODE REVIEW

**Comment by pali6** [comment]
What if we gave wifi components actual maximum range? 😳😳😳

**Comment by glassofmilk** [comment]
I feel like adding a max range to wifi components would be unfun and potentially break mechcomp setups that people have developed. I don't want to nerf something unless it's a problem

**Comment by glassofmilk** [comment]
Ok, I added a max range equal to what the packet sender on PDAs can do. I also added a minimum range because oops, I forgot to do that in the first place.

File: code/WorkInProgress/MechanicMadness.dm [hunk 1]
```diff
@@ -1679,7 +1679,8 @@ var/list/mechanics_telepads = new/list()
 
 	var/net_id = null //What is our ID on the network?
 	var/last_ping = 0
-	var/range = 0
+	var/range = 64
+	var/maxrange = 64
```
Comment (ZeWaka) [inline code/WorkInProgress/MechanicMadness.dm:1683]:
```suggestion
	var/const/maxrange = 64
```

Comment (ZeWaka) [reply code/WorkInProgress/MechanicMadness.dm:1683]: i think this should be a const, unless admin abuse, but still

**Comment by 2aa63b1b-7113-4f5a-8da3-7e8c95459c2c** [comment]
> I feel like adding a max range to wifi components would be unfun and potentially break mechcomp setups that people have developed. I don't want to nerf something unless it's a problem

Yes. This breaks pda chatrooms 👎 

If we are going to overhaul ranges, maybe allow players to easily build/deploy signal repeaters/range extenders?

**Comment by MarkNais** [comment]
It won't break PDA *groups* - PDA messaging still has infinite transmission range. 
Not sure what chatrooms you've made that are facilitated through MechComp.

**Comment by ZeWaka** [comment]
> It won't break PDA _groups_ - PDA messaging still has infinite transmission range.
> Not sure what chatrooms you've made that are facilitated through MechComp.

mechcomp pda chatrooms, often referred to as 'party lines'

**Pull request closed without merging**